worker.go

```diff
@@ -24,6 +24,7 @@ package ants
 
 import (
```

Comment:
You omitted the `worker_func.go`, please update it.

Reply:
got it